parquet-scrooge/src/main/java/parquet/scrooge/ScroogeStructConverter.java

```diff
@@ -98,8 +98,7 @@ private Class getCompanionClass(Class klass) {
       //each struct has a generated fieldInfos method to provide metadata to its fields
       try {
         Object r = klass.getMethod("fieldInfos").invoke(c);
-        Iterable<ThriftStructFieldInfo> a = JavaConversions$.MODULE$.asJavaIterable((scala.collection.Iterable<ThriftStructFieldInfo>)r);
-        return a;
+        return JavaConversions$.MODULE$.asJavaIterable((scala.collection.Iterable<ThriftStructFieldInfo>)r);
       } catch (ReflectiveOperationException e) {
```

Comment:
while you're in here, can you explicitly catch all the children of ReflectiveOperationException instead of ReflectiveOperationException? ReflectiveOperationException was added in java 7, so java 6 users get errors when using this (I heard from someone in the slack room)